numba/np/arraymath.py

```diff
@@ -894,25 +906,80 @@ def np_allclose_impl_array_array(a, b, rtol=1e-05, atol=1e-08,
             a_a, b_b = np.broadcast_arrays(a, b)
 
             for av, bv in np.nditer((a_a, b_b)):
-                if not _allclose_scalars(av.item(), bv.item(), rtol=rtol,
-                                         atol=atol, equal_nan=equal_nan):
+                if not _isclose_scalars(av.item(), bv.item(), rtol=rtol,
+                                        atol=atol, equal_nan=equal_nan):
                     return False
 
             return True
 
         return np_allclose_impl_array_array
 
 
-@overload(np.any)
-@overload_method(types.Array, "any")
-def np_any(a):
-    def flat_any(a):
-        for v in np.nditer(a):
-            if v.item():
-                return True
-        return False
+@overload(np.isclose)
+def np_isclose(a, b, rtol=1e-05, atol=1e-08, equal_nan=False):
```

Comment:
This file already contains an implementation of `np.isclose` in line 1191. Could you update the existing implementation? Or remove it in favor of this one?